p2p/pubsub/pubsub.go

```diff
@@ -14,6 +15,61 @@ import (
 	p2pmetrics "github.com/spacemeshos/go-spacemesh/p2p/metrics"
 )
 
+func init() {
+	// configure larger overlay parameters
+	pubsub.GossipSubD = 8
+	pubsub.GossipSubDscore = 6
+	pubsub.GossipSubDout = 3
+	pubsub.GossipSubDlo = 6
+	pubsub.GossipSubDhi = 12
+	pubsub.GossipSubDlazy = 12
+	pubsub.GossipSubDirectConnectInitialDelay = 30 * time.Second
+	pubsub.GossipSubIWantFollowupTime = 5 * time.Second
+	pubsub.GossipSubHistoryLength = 10
+	pubsub.GossipSubGossipFactor = 0.1
```

Comment:
is anything above changed from the default value? if so what was the reason?

Reply:
all these are different from the default. i copied this from filecoin. my rationale is that since i also copied their params for scoring, might as well use their default values.

https://github.com/filecoin-project/lotus/blob/d9311b200ca87d50243cd8deae525f62e78f26b4/node/modules/lp2p/pubsub.go#L28